src/ducks/notifications/Notification.js

```diff
@@ -54,12 +54,16 @@ class Notification {
 
       log('info', `Send notifications with category: ${attributes.category}`)
       const cozyClient = this.cozyClient
-      return cozyClient.fetchJSON('POST', '/notifications', {
+      await cozyClient.fetchJSON('POST', '/notifications', {
         data: {
           type: 'io.cozy.notifications',
           attributes
         }
       })
```

Comment:
Are we sure that the result was not used by the caller ?

Reply:
Yes